regressionTests.cmake

```diff
@@ -186,6 +186,13 @@ add_test_compareECLFiles(CASENAME numerical_aquifer_3d_1aqu
                          DIR aquifer-num
                          TEST_ARGS --tolerance-cnv=0.00003 --time-step-control=pid --linear-solver=cpr_trueimpes)
 
+add_test_compareECLFiles(CASENAME aquflux-01
+                         FILENAME AQUFLUX-01
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${rel_tol}
+                         DIR aquifers)
```

Comment:
Sorry, I didn't notice ealier.  You also need to hook this test case (`aquflux-01`) up to the automatic [data update script](https://github.com/OPM/opm-simulators/blob/49cca2ea621a715d1838d615f4f67a97d7ff316c/tests/update_reference_data.sh), typically between close to the [other aquifer tests](https://github.com/OPM/opm-simulators/blob/49cca2ea621a715d1838d615f4f67a97d7ff316c/tests/update_reference_data.sh#L82-L85).

Reply:
Oh, and by the way, the `CASENAME` should **probably** be `aquflux_01` instead of `aquflux-01` too.

Reply:
Sorry for that. I forgot the update script `again`.